### ISSUE TITLE
Contextify breaks on Node 4.2.1

### ISSUE THREAD

**Issue by jwulf** [getconnect/connect-js#61]
https://github.com/brianmcd/contextify/issues/180#issuecomment-145953429

Contextify doesn't build on Node 4.2.1, and apparently isn't needed for versions > 0.12.

**Comment by jwulf** [comment]
`$npm install connect-js-viz`
....

```
14 errors generated.
make: *** [Release/obj.target/contextify/src/contextify.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/jwulf/.node/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:270:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Darwin 15.0.0
gyp ERR! command "/usr/local/Cellar/node/4.2.1/bin/node" "/Users/jwulf/.node/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/jwulf/workspace/bgl-calculator-react/node_modules/contextify
gyp ERR! node -v v4.2.1
gyp ERR! node-gyp -v v3.0.3
gyp ERR! not ok 
npm ERR! Darwin 15.0.0
npm ERR! argv "/usr/local/Cellar/node/4.2.1/bin/node" "/Users/jwulf/.node/bin/npm" "install" "--save" "connect-js-viz"
npm ERR! node v4.2.1
npm ERR! npm  v3.3.8
npm ERR! code ELIFECYCLE

npm ERR! contextify@0.1.14 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
```

**Comment by chadedrupt** [comment]
Hi Joshua

I'll take a look at this again shortly.

**Comment by chadedrupt** [comment]
Fixed by e7a413323693bb407a486772f75a95cfa8b8d975
